lib/types/api.types.ts

```diff
@@ -125,14 +125,47 @@ export interface EditorLocaleSettings {
 export interface SharedEditorSDK {
   editor: {
     editorInterface: EditorInterface
+
+    /**
+     * Returns the current locale settings
+     *
+     * The locale setting can change. To always work with the latest settings, use `onLocaleSettingsChanged`.
+     */
+    getLocaleSettings(): EditorLocaleSettings
+
+    /**
+     * Subscribes to changes of the editor's locale settings
+     *
+     * @param callback Function that is called every time the locale settings change. Called immidiately with the current setting.
+     * @returns Function to unsubscribe. `callback` won't be called anymore.
+     */
     onLocaleSettingsChanged: (
       callback: (localeSettings: EditorLocaleSettings) => void
     ) => () => void
-    onShowDisabledFieldsChanged: (callback: (showDisabledFields: boolean) => any) => () => void
+
+    /**
+     * Returns whether or not disabled fields are displayed
+     *
+     * This setting can change. To always work with the latest settings, use `onShowDisabledFieldsChanged`.
+     */
+    getShowDisabledFields(): boolean
+
+    /**
+     * Subscribes to changes of whether or not disabled fields are displayed
```

Comment:
**Just an FYI:** The editor feature was rebranded to "Show hidden fields" recently (see [changelog entry](https://www.contentful.com/developers/changelog/#hidden-entry-editor-fields))

Would it make sense to reflect that also in the SDK and deprecate the old functions to avoid confusion?

Reply:
I already mixed the two types of "disabled" during the implementation of this PR. So, we will deprecated `onIsDisabledChanged` and ho for "hidden"